src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -206,11 +206,16 @@ public new static RevitDynamoModel Start(IRevitStartConfiguration configuration)
 
         public override void PostTraceReconciliation(Dictionary<Guid, List<ISerializable>> orphanedSerializables)
         {
-            var orphanedIds =
+            var orphanedIds = new List<int>();
+            var serializables =
                 orphanedSerializables
-                .SelectMany(kvp=>kvp.Value)
-                .Cast<SerializableId>()
-                .Select(sid => sid.IntID).ToList();
+                .SelectMany(kvp => kvp.Value)
+                .Where(x => x is ISerializable);
+
+            orphanedIds.AddRange(serializables.Where(x => x is SerializableId).
+                Cast<SerializableId>().Select(sid => sid.IntID));
+            orphanedIds.AddRange(serializables.Where(x => x is MultipleSerializableId).
```

Comment:
Should MultipleSerializableId be derived from SerializableId?? What if we introduce a new ISerializable class in future that are not derived from either SerializableId and MultipleSerializableId??